swagger/swagger.yml

```diff
@@ -0,0 +1,86 @@
+produces:
+- application/json
+schemes:
+- http
+swagger: "2.0"
+consumes:
+- application/json
+info:
+  title: Swagger
+  version: 0.1.0
+paths: 
+  /accounts:
+    post:
+      tags: 
+        - account
+      operationId: add_Account
+      parameters:
+        - name: body
+          required: true
+          in: body
+          schema:
+            $ref: "#/definitions/account"
+      responses:
+        201:
+          description: Created
+        400:
+          description: Name should be atleast 3 characters long
+          schema:
+            $ref: "#/definitions/error"
+        default:
+          description: error
+          schema:
+            $ref: "#/definitions/error"
+definitions:
+  account:
+    type: object
+    required: 
+      - user_name
+    properties:
+      id:
```

Comment:
set readonly true for id, user_name, created_at, updated_at

Reply:
Done please check